controllers/clusterca_to_secret.go

```diff
@@ -0,0 +1,190 @@
+package controllers
+
+import (
+	"context"
+	"encoding/hex"
+	"fmt"
+	"time"
+
+	"github.com/go-logr/logr"
+	v1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/api/errors"
+	v12 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/types"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+	"sigs.k8s.io/controller-runtime/pkg/log"
+)
+
+const (
+	Namespace  = "global-trust-certificates"
```

Comment:
We'll want to make these configurable, maybe through command-line parameters. Also, wondering do we handle the case where the cluster doesn't have the indicated namespace created.

Reply:
So the plan is to edit the start up command to feed in Namespace and secret name per cluster?

Second question, no there isnt a check for namespace. Will add that in.